### PR TITLE
New presence for Radio Garden

### PR DESCRIPTION
**Pull request by tepiloxtl**
Proof of working presence. Includes radiostation name, location and listening time elapsed

image

### CODE REVIEW

File: Radio Garden/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "id": "085733181730332672"
+  },
+  "url": "radio.garden",
+  "description": {
+    "en": "By bringing distant voices close, radio connects people and places. Radio Garden allows listeners to tune into live radio across entire globe."
+  },
+  "service": "Radio Garden",
+  "version": "1.0",
+  "logo": "https://vikavo.lt/premidpresence/radiogardenglobe.webp",
+  "thumbnail": "https://vikavo.lt/premidpresence/radiogardenthumb.png",
```
Comment (Bas950) [inline Radio Garden/dist/metadata.json]: upload to imgur

File: Radio Garden/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "Tepiloxtl",
+    "id": "085733181730332672"
+  },
+  "url": "radio.garden",
+  "description": {
+    "en": "By bringing distant voices close, radio connects people and places. Radio Garden allows listeners to tune into live radio across entire globe."
+  },
+  "service": "Radio Garden",
+  "version": "1.0",
+  "logo": "https://vikavo.lt/premidpresence/radiogardenglobe.webp",
```
Comment (Alanexei) [inline Radio Garden/dist/metadata.json]: You own vikavo or have permission to make changes right?

**Comment by Fruxh** [comment]
@Mergifyio refresh

File: Radio Garden/dist/metadata.json (hunk 2 above)
Comment (tepiloxtl) [reply Radio Garden/dist/metadata.json]: I own vikavo.lt. I can reupload to imgur though per @Bas950 request.

Comment (doomlerd) [reply Radio Garden/dist/metadata.json]: It's fine.

**Pull request merged**